### ISSUE TITLE
Calvaryman & Horseman unit lines renaming proposal

### ISSUE THREAD

**Issue by sheepo99** [wesnoth/wesnoth#6404]
I've been working on the translation of Wesnoth to my language and I've noticed that some units appear to have somewhat repetitive or confusing names which not only difficult translation but also bring consistency issues to the English original. One such example are the Horseman and Cavalryman unit lines. If we compare the Horseman mainline to the Cavalryman line this quickly becomes evident:

![image](https://user-images.githubusercontent.com/3621746/147585431-32c7d04d-d843-4770-a786-fe49252bc808.png)

"Horseman" "Cavalryman" and "Cavalier" are all effectively synonyms or semantically similar terms which can make distinction difficult name-wise and also don't reflect the unique characteristics and lore behind each of these units very well. So I would like to propose three of these units to be renamed to more adequate terms.

**The Horseman Line**

The idea here is that the Horseman line mainly represents knighted gentry whose combat techniques go hand-in-hand with the chivalric code. The "horseman" instead becomes an aspirant to a knight: a Squire. The line is then renamed as follows:

_Squire - Knight - Grand Knight_

Renaming the Horseman to squire maintains the theme of knighthood while giving it a better sense of progression.

**The Cavalryman line**

The cavalryman line's theme seems to be mounted soldiers who do not necessarily adhere to the chilvaric code. Historically, these warriors were known as Men-at-Arms. **While technically all knights were men-at-arms, not all men-at-arms were knights**. My proposal to rename this line is as follows:

_Man-at-Arms - Dragoon - Cuirassier_

The rationale here is that this naming draws a parallel with the shift of late medieval knighted gentry towards modern cavalry corps, starting around the 15th century. And in effect, Men-at-Arms did evolve into Dragoons and Cuirassiers. This solution has already been adopted in some of the translations, if we take a peek at the German and French unit names:

![image](https://user-images.githubusercontent.com/3621746/147587652-0e149a20-7513-42bc-ba2e-81b9c5bea697.png)

**Conclusion**

All in all, I think the adoption of these terms would look nicer and less confusing in game, and would make it easier to translate while making English consistent with already existing translations.

**Sources:**
https://en.wikipedia.org/wiki/Man-at-arms
https://en.wikipedia.org/wiki/Squire
https://en.wikipedia.org/wiki/Cuirassier

**Comment by knyghtmare** [comment]
I don't think this would be a good idea as an ID-change will cause breakage of existing survival scenarios, campaigns (saves/replays), and 50-100 existing UMC.

Also, a level 3 unit called "Master at Arms" exists and it can be confused with the proposed level 1 rename.

**Comment by sheepo99** [comment]
Unit names have however been changed in the past. Obviously sometime save games might break, but I don't think that's a good argument to prevent improvements to unit naming - or anything else that might interfere with stability. Scenarios, campaigns, etc, can be updated for this purpose too.

Another idea is to make a more comprehensive list of unit names that could use an update and change them all at once.

**Comment by sheepo99** [comment]
Also can we simply not keep the same unit IDs while changing the name labels instead?

**Comment by CelticMinstrel** [comment]
I don't think all the new proposed names are good, either. Squire is fine – it does make a lot of sense for the predecessor to the knight to be a squire. But Cuirassier doesn't fit at all – it's too modern of a term to apply to a medieval horseman (note that a historical cuirassier would've been armed with a sword and _pistol_). As for man-at-arms (plural men-at-arms)… I'm mostly neutral on it.

> Also can we simply not keep the same unit IDs while changing the name labels instead?

This is a long-standing problem in Wesnoth, that units by convention use their name as the ID, so changing the name means changing the ID. It would certainly be confusing if the ID was `Cavalryman` and the name was `Man-at-arms`, right? If we wanted to de-link the name from the ID, I think we'd need to change _every_ ID to be something that looks more like an internal detail. Just as an example, the horseman line could become "knight1, knight2, knight3" and the cavalryman line could become "horseman1, horseman2, horseman3".

-----

All that aside, if you find this naming strategy to make things easier to translate, then feel free to write the translations _as if_ this naming strategy had been applied. Translation is an art, not a science. As long as your translations give a similar impression as the original English terms, then it's fine. If you want to translate "Horseman" as "Écuyer" and "Cavalryman" as "Hussard", then that's totally fine, even though those French words don't directly translate to the original English.

(I assume the language you're translating to isn't French, and I'm also not 100% sure that's the correct way to translate "squire" into French, but it was just an example.)

**Comment by CelticMinstrel** [comment]
A couple more responses…

> Also, a level 3 unit called "Master at Arms" exists and it can be confused with the proposed level 1 rename.

I don't think it's all that confusing? I see a clear difference between a "master-at-arms" and a "man-at-arms". Note though that "Master at Arms" is incorrect (it should be hyphenated), so maybe that unit should also be renamed…

> Another idea is to make a more comprehensive list of unit names that could use an update and change them all at once.

It might actually be a good idea to do this at some point… though I don't think there would be a huge number of them.

**Comment by sheepo99** [comment]
> I don't think all the new proposed names are good, either. Squire is fine – it does make a lot of sense for the predecessor to the knight to be a squire. But Cuirassier doesn't fit at all – it's too modern of a term to apply to a medieval horseman (note that a historical cuirassier would've been armed with a sword and _pistol_). As for man-at-arms (plural men-at-arms)… I'm mostly neutral on it.

This is the same thing for the Dragoon though. They were mounted infantry/ranged cavalry from the 16th century. I picked Cuirassier precisely because of that, and so did the French and German translators. Source: https://en.wikipedia.org/wiki/Dragoon

>This is a long-standing problem in Wesnoth, that units by convention use their name as the ID, so changing the name means changing the ID. It would certainly be confusing if the ID was Cavalryman and the name was Man-at-arms, right? If we wanted to de-link the name from the ID, I think we'd need to change every ID to be something that looks more like an internal detail. Just as an example, the horseman line could become "knight1, knight2, knight3" and the cavalryman line could become "horseman1, horseman2, horseman3".

I think doing so would be very positive because it would allow for greater flexibility to fix or tweak anything pertaining naming aspects. I can understand how this might seem like a lot of work for the sake of details, but having worked as a proofreader and translator for the gaming industry, I can tell you language consistency and polish does have an important effect on how players perceive the experience of playing the game, which is why I believe matters like these shouldn't simply be ignored.

**Comment by CelticMinstrel** [comment]
> This is the same thing for the Dragoon though. They were mounted infantry/ranged cavalry from the 16th century. I picked Cuirassier precisely because of that, and so did the French and German translators.

Fair enough… Cuirassier is also far less recognizable though. I've known what a Dragoon is for a long time, at least in a basic sense, but I had to look up Cuirassier.

**Comment by sheepo99** [comment]
> Fair enough… Cuirassier is also far less recognizable though. I've known what a Dragoon is for a long time, at least in a basic sense, but I had to look up Cuirassier.

I'd say they're about the same for anyone who read a bit about post-medieval military history or is into period gaming, although the Dragoon name is used more frequently in fantasy themes, such as the soldier class in Final Fantasy. And again the same translation is being used in French, German, Dutch, Russian, Spanish, and Italian, which clearly points towards a consensus in meaning and concordance between languages, along with general familiarity with the term. This is also why I think it's worth to use it for the English name.

![image](https://user-images.githubusercontent.com/3621746/147612568-34012cc2-66a6-497c-a1a6-5a14c3653928.png)

**Comment by CelticMinstrel** [comment]
Just because it's a good choice in 7 other languages, that doesn't mean it's a good choice in English.

**Comment by sheepo99** [comment]
I do believe it to be a good choice in English, and you can take my word as a language professional. It is nevertheless more consistent than having Cavalryman and Cavalier together in the same unit tier.

**Comment by CelticMinstrel** [comment]
I really don't think there's anything wrong with the terms "cavalier" and "cavalryman". It's true that it's weird to have "cavalryman" and "cavalier" in the same unit line, but I think changing just one of them would be sufficient. For example, "man-at-arms – dragoon – cavalier" seems fine to me. (If anything I feel like dragoon should've been the L3, but whatever.)

**Comment by sheepo99** [comment]
> I really don't think there's anything wrong with the terms "cavalier" and "cavalryman". It's true that it's weird to have both in the same unit line, but I think changing just one of them would be sufficient. For example, "man-at-arms – dragoon – cavalier" seems fine to me. (If anything I feel like dragoon should've been the L3, but whatever.)

This is fine to an extent, but there's an issue with the term "cavalier" too. While "cavalryman" is yet another generic term for "guy who rides a horse", "cavalier" is a name generally associated with the last remainders of chivalric nobility during the English Civil War: https://en.wikipedia.org/wiki/Cavalier

So a Cavalier would be a more logical evolution for a Knight, although I think Grand Knight is just fine as is. The key difference that is useful to emphasize between both unit lines is the one between high ranking military nobility focusing on lance and charge tactics (traditional knights) and low-ranking or non-noble professional cavalry focusing on multi-weapon combat (dragoons). The name choices I've listed do this very well, and beyond that, draw a nice little parallel with real-life military history which has some educational value for younger players.

More importantly, though, I would like to know what can be done to make unit name changes a bit easier in the future, given the internal structure issues mentioned above. This is because there's a fair amount of other unit names that are problematic and could use a review in the future.

**Comment by CelticMinstrel** [comment]
Even if you don't like "cavalier", you can use "cavalryman" instead. Admittedly, that probably works better if you rearrange the order, but still – there's no need to bring in something relatively obscure like "cuirassier".

**Comment by sheepo99** [comment]
This is ok for the time being. Can we agree to change Horseman to Squire and Cavalryman to Man-at-Arms as a first step?

**Comment by Wedge009** [comment]
I remember this being discussed when #6071 was first submitted. I apologise for not reading through the full discussion here but from my opinion: I think what translators have done for other languages can be a good indication of what may actually fit better for the original English. Semantically, I don't object to the suggestions. What I do wonder is the scope of the work involved: if we just change the unit names (and not the IDs) it does ease the burden on the number of changes to make this proposal work but then there's a digression between the ID and naming of units which can lead to confusion if it's not documented well in the unit code. Then there's the potential for long-time players to be thrown off by the changes, though I suppose that shouldn't necessarily be an impediment to making the change otherwise no changes would be possible if we took this as a blocker.

I think this change - if it does go ahead - should only go to master and leave 1.16 as-is. A major update like 1.18.0 should allow for changes such as this kind of renaming.

**Comment by sheepo99** [comment]
I agree a major update to allow for flexible changing of unit names would be a desirable consequence and, as mentioned before, this could go along an overall review of all unit names and other in-universe nomenclatures. I would personally love helping with this, since anything regarding textual consistency and QC is my specialty!
